addons/account/wizard/accrued_orders.py

```diff
@@ -244,6 +244,8 @@ def _ellipsis(string, size):
             'date': self.date,
             'line_ids': move_lines,
         }
+        if len(orders) == 1 and orders.company_id.currency_id != orders.currency_id:
+            move_vals['currency_id'] = orders.currency_id.id
```

Comment:
You can't do that :sweat: So in case of multiple orders, the bug will respawn? Also, `currency_id` is there only for business models (payments, invoices)... That's weird to force a value for it for a miscellaneous operations. Usually, you need to base your computations on amls's currency_id instead

Reply:
hello @smetl can you check this alternative fix